src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+/* eslint-disable no-unused-vars */
```

Comment:
As suggested by the previous reviewer, kindly enable all linters as helps you have a clean and standardized code that helps you regarding best practices and DRY method 😄